Installing-and-Configuring/GPG.md

```diff
@@ -9,9 +9,10 @@ nav_exclude: true
 Generating a new card
 ===
 
-NOTE: gpg 2.1.21 is required to create 4096 bits keys, which is not provided with heads for the moment.
+NOTE: gpg 2.1.21 is required to create 4096 bits keys, which is not provided
```

Comment:
False. Delete, is merged since a looooong while

Reply:
Though putting it in 4096 is manual. 3072 is default. Debate under https://github.com/osresearch/heads/issues/831